### PR TITLE
Fix some comments in websockets example

### PR DESCRIPTION
**Pull request by FrankReh**
A few comments in the websockets example weren't complete. A few had trivial typos. And one comment change with a gender-neutral seems to be more idiomatic with tokio comments.

### CODE REVIEW

**Comment by FrankReh** [comment]
Let me know if you would accept even more trivial changes to the comments to fix capitalization and punctuation, just to make them consistent for the two rust source files.

File: examples/websockets/src/main.rs [hunk 1]
```diff
@@ -131,7 +134,7 @@ async fn handle_socket(mut socket: WebSocket, who: SocketAddr) {
 
     // Since each client gets individual statemachine, we can pause handling
     // when necessary to wait for some external event (in this case illustrated by sleeping).
-    // Waiting for this client to finish getting his greetings does not prevent other clients form
+    // Waiting for this client to finish getting their greetings does not prevent other clients from
```
Comment (jplatte) [inline examples/websockets/src/main.rs:137]: I think this should be "its".

**Comment by FrankReh** [comment]
Also a handful of clippy warnings.

File: examples/websockets/src/main.rs [hunk 2]
```diff
@@ -63,8 +67,7 @@ async fn main() {
                     )
                 }),
         )
-        // routes are matched from bottom to top, so we have to put `nest` at the
-        // top since it matches all routes
+        // routes are matched from bottom to top
```
Comment (davidpdrsn) [inline examples/websockets/src/main.rs:70]: This is no longer true actually and haven't been for a while. So we can just remove this comment.

```suggestion
```

**Review by davidpdrsn — APPROVED** [review]
Thanks!

**Pull request merged**